src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const table = document.getElementsByTagName('table')[0];
```

Comment:
```suggestion
const table = document.querySelector('table');
```
you can simplify that